src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2477,3 +2481,70 @@ def _aks_mesh_update(
         return None
 
     return aks_update_decorator.update_mc(mc)
+
+def _which(binary):
+    path_var = os.getenv('PATH')
+    if platform.system() == 'Windows':
+        binary = binary + '.exe'
+        parts = path_var.split(';')
+    else:
+        parts = path_var.split(':')
+
+    for part in parts:
+        bin_path = os.path.join(part, binary)
+        if os.path.exists(bin_path) and os.path.isfile(bin_path) and os.access(bin_path, os.X_OK):
+            return bin_path
+
+    return None
+
+def _process_message(message):
+    result = message.split("\n")
+    for line in result:
+        print(line)
+
+def aks_check_network_vmss(cmd, 
+                      client, 
+                      resource_group, 
+                      cluster_name, 
+                      location,
+                      node_name, 
+                      no_wait=False, 
+                      aks_custom_headers=None):
+    # Get a random node if node_name is not specified
+    if not node_name:
+        if not _which("kubectl"):
+            raise ValidationError("Can not find kubectl executable in PATH")
+        
+        fd, browse_path = tempfile.mkstemp()
+        try:
+            aks_get_credentials(cmd, client, resource_group, cluster_name, admin=False, path=browse_path)
+
+            cmd = f"kubectl --kubeconfig {browse_path} get nodes"
+            output = subprocess.Popen(cmd, shell=True, stderr=subprocess.PIPE)
+        except subprocess.CalledProcessError as ex:
+            raise ValidationError("Can not get node name from cluster: {}".format(ex))
+        if output:
+            result, _ = output.communicate()
+            cluster_info = json.loads(result)
+            node_name = str(cluster_info["items"][0]["metadata"]["name"])
+        else:
+            raise ValidationError("Failed to get node name from cluster")
+    index = node_name.find("vmss")
+    vmss_name = node_name[0:index+4]
+    instance_id = int(node_name[index+4:])
+    node_resource_group = "MC_{0}_{1}_{2}".format(resource_group, cluster_name, location)
+    print("Check parameters vmss_name: {0}, instance_id: {1}, node_resource_group: {2}".format(vmss_name, instance_id, node_resource_group))
+
+    from azure.cli.command_modules.vm.custom import vmss_run_command_invoke
+    try:
+        command_result_poller = vmss_run_command_invoke(cmd, node_resource_group, vmss_name, command_id="RunShellScript", scripts="bash /usr/local/bin/check-outbound-network.sh", instance_id=instance_id)
```

Comment:
It's hard to re-use this func as it doesn't load the models with necessary kwargs (missing resource_type and operation_group) for outer func to use.

Could call the operation by yourself. May refer to commit [#88db6ef](https://github.com/FumingZhang/azure-cli-extensions/commit/88db6ef7bea0416fdc61fd19ca409780ed9d47b8).